lib/inputs/java/static.ts

```diff
@@ -3,11 +3,13 @@ import { ExtractAction } from "../../extractor/types";
 import { streamToBuffer } from "../../stream-utils";
 
 const ignoredPaths = ["/usr/lib", "gradle/cache"];
+const javaArchiveFileFormats = [".jar", ".war"];
 
 function filePathMatches(filePath: string): boolean {
   const dirName = path.dirname(filePath);
+  const fileExtension = filePath.slice(-4);
```

Comment:
should we consider breaking the the file path by ".", and looking at the last element of the array? I might be misunderstanding, but what about the option that the file ends with something like ".warwar"? I know this might actually be an option, but maybe it'd be better to avoid such case?

Reply:
splitting by "." is a valid option as well, but ".warwar" is not something we want to scan, and will not pass the condition in line 12

Reply:
You are right, and anyways I don't feel strongly about this.